instrumentation/runtime-telemetry-jfr/javaagent/src/test/groovy/io/opentelemetry/instrumentation/javaagent/runtimetelemetryjfr/JfrRuntimeMetricsTest.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.javaagent.runtimetelemetryjfr
+
+import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
+import spock.util.concurrent.PollingConditions
+
+class JfrRuntimeMetricsTest extends AgentInstrumentationSpecification {
```

Comment:
I don't think we want to introduce new groovy tests.

Reply:
Ok I can remove these. Should I change this to use JUnit then?

Reply:
Yes, junit + assertj + java is preferred (I think 🙂).